platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/constants/Style.java

```diff
@@ -17,39 +20,196 @@
 public class Style {
 
     /**
+     * Mapbox Streets: A complete basemap, perfect for incorporating your own data.
+     */
+    private static final String MAPBOX_STREETS_BASE = "mapbox://styles/mapbox/streets-v%d";
+    /**
+     * Outdoors: A rugged style that emphasizes physical terrain and outdoor activities.
+     */
+    private static final String OUTDOORS_BASE = "mapbox://styles/mapbox/outdoors-v%d";
+    /**
+     * Light: Subtle light backdrop for data visualizations.
+     */
+    private static final String LIGHT_BASE = "mapbox://styles/mapbox/light-v%d";
+    /**
+     * Dark: Subtle dark backdrop for data visualizations.
+     */
+    private static final String DARK_BASE = "mapbox://styles/mapbox/dark-v%d";
+    /**
+     * Satellite: A beautiful global satellite and aerial imagery layer.
+     */
+    private static final String SATELLITE_BASE = "mapbox://styles/mapbox/satellite-v%d";
+    /**
+     * Satellite Streets: Global satellite and aerial imagery with unobtrusive labels.
+     */
+    private static final String SATELLITE_STREETS_BASE = "mapbox://styles/mapbox/satellite-hybrid-v%d";
+
+    /**
+     * Get versioned url of Mapbox streets style.
+     * <p>
+     * <ul>
+     * <li>Current default version is 9.</li>
+     * </ul
+     * </p>
+     * <p>
+     * More information on the Mapbox styles API can be found on https://www.mapbox.com/api-documentation/#styles
+     * </p>
+     *
+     * @param version the version of the style.
+     * @return uri to load style from
+     */
+    public static String getMapboxStreetsUrl(@IntRange(from = 1, to = 9) int version) {
```

Comment:
We shouldn't be setting an upper bound at all. Half the reason for versioning this URL API is so that we can decouple SDK releases from style releases. If we need to issue a version 10 style a week later, that means no one can use that version until we release a new SDK version.